### ISSUE TITLE
'list' object has no attribute 'keys'

### ISSUE THREAD

**Issue by seeebek** [Marginal/EDMarketConnector#15]
After the last update there is this error:
![error](https://cloud.githubusercontent.com/assets/1104406/8572488/3a66291a-258d-11e5-85b5-16b8974f7ce2.png)

```
Traceback (most recent call last):
  File "C:\Users\Sebastian\Downloads\EDMarketConnector-master (1)\EDMarketConnector-master\EDMarketConnector.py", line 240, in getandsend
    eddn.export(data)
  File "C:\Users\Sebastian\Downloads\EDMarketConnector-master (1)\EDMarketConnector-master\eddn.py", line 100, in export
    'ships'       : [ship_map.get(ship,ship) for ship in (data['lastStarport']['ships'].get('shipyard_list') or {}).keys() + (data['lastStarport']['ships'].get('unavailable_list') or {}).keys()],
AttributeError: 'list' object has no attribute 'keys'
```

**Comment by Marginal** [comment]
Dealing with the Companion API is like playing Twenty Questions!
I suspect the problem is with 'unavailable_list' - I've only ever seen it empty.

Please mail me the .json file that was produced when you ran from source in order to obtain that traceback.

**Comment by seeebek** [comment]
You have mail ;-)

**Comment by Marginal** [comment]
Thanks. Fixed in [Release 1.61](https://github.com/Marginal/EDMarketConnector/releases/latest).

**Comment by Marginal** [comment]
> I think the error happens on stations without shipyards.

I think this is a separate issue. I've raised Issue #16.
